core/lib/dal/migrations/20240805144000_tee_proofs_reorg.up.sql

```diff
@@ -0,0 +1,24 @@
+DROP INDEX IF EXISTS idx_tee_proof_generation_details_status_prover_taken_at;
+CREATE TYPE tee_proofs_job_status AS ENUM ('ReadyToBeProven', 'PickedByProver', 'Generated', 'Skipped');
```

Comment:
Migration is breaking in a sense that we should coordinate applying it and deploying services. If we decide to go with it (which I would like to avoid), then please make sure you disable service, apply migration, deploy updated service.

Reply:
I've radically simplified this file, so please re-review. Thanks!